Headers/DebugServer2/HardwareBreakpointManager.h

```diff
@@ -23,8 +23,6 @@
 
 namespace ds2 {
 
-static const int kMaxHWStoppoints = 32;
```

Comment:
I think this commit is good except for the outdated commit message. I'll rewrite the commit message before committing.